Generate.py

```diff
@@ -133,7 +134,10 @@ def main(args=None) -> tuple[argparse.Namespace, int]:
                     if yaml is None:
                         logging.warning(f"Ignoring empty yaml document #{doc_idx + 1} in {fname}")
                     else:
-                        weights_for_file.append(yaml)
+                        quantity = yaml.get("quantity", 1) if allow_quantity else 1
```

Comment:
I think this should error on a yaml that has a quantity other than 1 defined if the setting is not allowed

Reply:
Yeah